### PR TITLE
Fix changelog before release

### PR DESCRIPTION
**Pull request by confluence**
I have added what I think are all the relevant entries that were missing from the changelog. I have also moved an entry that I incorrectly added to the previous release, and changed the title of the current release section to `Unreleased` -- this should only be changed to the release version during the release.

I have omitted changes which affect only unit tests, compilation or our CI, since I don't think these are user-facing, but I did add a catch-all entry for all the ASAN fixes and memory leak fixes. I have also omitted fixes to changes within the same release (these should be considered part of the original change). Please let me know if you think I've missed something that should be added.

### CODE REVIEW

**Comment by pford** [comment]
Should the DS9 import bug be added, or is it internal only (ASAN)?  If so, it was missed.

* Fixed DS9 import bug on MacOS ([#1066](https://github.com/CARTAvis/carta-backend/issues/1066)), ([#1096](https://github.com/CARTAvis/carta-backend/issues/1096)).

I guess this is part of the general ASAN entry.

**Review by pford — APPROVED** [review]
All my stuff is in.  Thanks @confluence !

**Review by jolopezl — CHANGES_REQUESTED** [review]
Thanks! I am sorry for not adding this when due. I am missing the performance improvements in the image cache (#967, #1034).

**Comment by confluence** [comment]
> I am missing the performance improvements in the image cache (#967, #1034).

I was on the fence about that, but you're right; if it's a performance optimisation, that's user-facing. I'll put it in.

**Comment by jolopezl** [comment]
> > I am missing the performance improvements in the image cache (#967, #1034).
> 
> I was on the fence about that, but you're right; if it's a performance optimisation, that's user-facing. I'll put it in.

Thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,23 +4,45 @@ All notable changes to this project will be documented in this file.
...
+* Removed gRPC service and moved scripting interface to HTTP server ([#1022](https://github.com/CARTAvis/carta-backend/pull/1022)).
+* Added more fine-grained commandline flags to enable and disable functions of the HTTP server.
+* Optimised performance of image data cache ([#967](https://github.com/CARTAvis/carta-backend/issues/967)).
+* Added exit on timeout flag to Linux desktop launcher ([#989](https://github.com/CARTAvis/carta-backend/issues/989)).
+* Separated stdout and stderr logging ([#869](https://github.com/CARTAvis/carta-backend/issues/869)).
 
 ### Added
+* Added support for creating CASA LEL images dynamically ([#655](https://github.com/CARTAvis/carta-backend/issues/665)).
+* Added spatial and spectral range information to the file browser ([#845](https://github.com/CARTAvis/carta-backend/issues/845)).
 * Added computed stokes images and supported the analysis (profiles, contours, statistics, etc.) of them ([#433](https://github.com/CARTAvis/carta-backend/issues/433)).
```
Comment (kswang1029) [inline CHANGELOG.md]: stokes -> Stokes

**Review by jolopezl — APPROVED** [review]

**Pull request merged**